authbridge/demos/finance-sparc/scripts/show-verdicts.py

```
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+"""Print the SPARC plugin verdicts from a session snapshot (read on stdin)."""
+import json
+import sys
+
+try:
+    data = json.load(sys.stdin)
+except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)